template.md

```diff
@@ -0,0 +1,305 @@
+
+### Internal Links
+
+To link to a header in the same Markdown file, view the source of the published article, find the id of the head (for example, `id="blockquote"`), and link using # + id (for example, `#blockquote`).
+
+- Example: [Blockquotes](#blockquote)
+
+To link to a Markdown file in the same repo, use [relative links](https://www.w3.org/TR/WD-html40-970917/htmlweb.html#h-5.1.2), including the ".md" at the end of the filename.
+
+- Example: [Readme file](readme.md)
+- Example: [Welcome to .NET](./docs/welcome.md)
+
+To link to a header in a Markdown file in the same repo, use relative linking + hashtag linking.
+
+- Example: [.NET Community](./docs/welcome.md#community)
+
+### External Links
+
+To link to an external file, use the full URL as the link.
+
+- Example: [GitHub](http://www.github.com)
+
+If a URL appears in a Markdown file, it will be transformed into a clickable link.
+
+- Example: http://www.github.com
+
+## Lists
+
+### Ordered lists
+
+1. This 
+1. Is
+1. An
+1. Ordered
+1. List  
+
+
+#### Ordered list with an embedded list
+
+1. Here
+1. comes
+1. an
+1. embedded
+    1. Miss Scarlett
+    1. Professor Plum
+1. ordered
+1. list
+
+
+### Unordered Lists
+
+- This
+- is
+- a
+- bulleted
+- list
+
+
+##### Unordered list with an embedded list
+
+- This 
+- bulleted 
+- list
+    - Mrs. Peacock
+    - Mr. Green
+- contains  
+- other
+    1. Colonel Mustard
+    1. Mrs. White
+- lists
+
+
+## Horizontal rule
+
+---
+
+## Tables
+
+| Tables        | Are           | Cool  |
+| ------------- |:-------------:| -----:|
+| col 3 is      | right-aligned | $1600 |
+| col 2 is      | centered      |   $12 |
+| col 1 is default | left-aligned     |    $1 |
+
+
+## Code
+
+### Generic codeblock
+
+Indent code four spaces for generic codeblock coding.
+
+    function fancyAlert(arg) {
+      if(arg) {
+        $.docs({div:'#foo'})
+      }
+    }
+
+
+### Codeblocks with language identifier
+
+Use three backticks (&#96;&#96;&#96;) + a language ID to apply language-specific color coding to a code block.  Here is the entire list of [GitHub Flavored Markdown (GFM) language IDs](https://github.com/jmm/gfm-lang-ids/wiki/GitHub-Flavored-Markdown-(GFM)-language-IDs).
+
+##### C&#9839;
+
+```c#
+using System;
+namespace HelloWorld
+{
+    class Hello 
+    {
+        static void Main() 
+        {
+            Console.WriteLine("Hello World!");
+
+            // Keep the console window open in debug mode.
+            Console.WriteLine("Press any key to exit.");
+            Console.ReadKey();
+        }
+    }
+}
+```
+#### Python
+
+```python
+friends = ['john', 'pat', 'gary', 'michael']
+for i, name in enumerate(friends):
+    print "iteration {iteration} is {name}".format(iteration=i, name=name)
+```
+#### PowerShell
+
+```powershell
+Clear-Host
+$Directory = "C:\Windows\"
+$Files = Get-Childitem $Directory -recurse -Include *.log `
+-ErrorAction SilentlyContinue
+```
+
+### Inline code
+
+Use backticks (&#96;) for `inline code`.
```

Comment:
Those are good points plus the capitalization one. We need to update here and in the main template: https://github.com/Microsoft/Docs/blob/master/template.md
Hopefully, in the future, we can just link there, but for now I've created one here because there are a lot of things that don't apply to us.